closure/goog/soy/soy_testhelper.js

```diff
@@ -64,44 +72,79 @@ function makeSanitizedContent(content, kind) {
 var example = {};
 
 
-example.textNodeTemplate = function(opt_data, opt_sb, opt_injectedData) {
-  assertNotNull(opt_data);
-  assertNotUndefined(opt_data);
-  return goog.string.htmlEscape(opt_data.name);
+/**
+ * @param {{name: string}} data
+ * @param {null=} opt_sb
```

Comment:
These are Soy templates.
`soy_test.js` uses:
`goog.soy.renderElement`: https://github.com/google/closure-library/blob/master/closure/goog/soy/soy.js#L67-L68
`goog.soy.renderAsFragment`: https://github.com/google/closure-library/blob/master/closure/goog/soy/soy.js#L91-L92
I determined the types from the signatures there.